packages/tsconfig-reference/copy/ja/options/allowJs.md

```diff
@@ -3,16 +3,16 @@ display: "Allow JS"
 oneline: "Let TS include .JS files in imports"
 ---
 
-`.ts`、`.tsx`ファイルだけでなく、JavaScriptファイルをプロジェクトへインポートできるようにします。例えば、次のJSファイルを:
+`.ts`、`.tsx`ファイルだけでなく、JavaScript ファイルをプロジェクトへインポートできるようにします。例えば、次の JS ファイルを:
```

Comment:
I've asked in the Japanese localization channel about these changes

Reply:
No problem 😄  Thanks for installing Prettier.